### DISCUSSION TITLE
inpaint feature form fooocus

### DISCUSSION THREAD

**Discussion by kalle07** [mcmonkeyprojects/SwarmUI#373] [Ideas]
out of 5 inpaint Ai softwares I found this one the most comfortable and fastest and that with less VRAM, even 2000pix images with SDXL use only 8GB
i know at the moment its not FLUX ready
maybe you can take a look  ;)

**Comment by mcmonkey4eva** [comment]
what?

**Comment by kalle07** [comment]
https://github.com/lllyasviel/Fooocus
